hokusai/test.yml

```diff
@@ -8,6 +8,4 @@ services:
     environment:
       - CI=true
       - CIRCLE_NODE_INDEX
-      - CDN_PRODUCTION_URL
```

Comment:
These could me moved to `build.yml` to be available at build time - I am not sure hardcoding a production URL in the Dockerfile is the best approach